app/furniture/marketplace/management_component.html.erb

```diff
@@ -6,18 +6,19 @@
     <%= marketplace.room.name %>
   </p>
 </div>
-<%= render onboarding_component %>
-<%= render CardComponent.new(classes: "mt-3") do |card| %>
-  <%= content %>
-
-  <% card.with_footer do %>
-    <nav class="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 gap-3">
-      <%= render button({child: :stripe_account}, icon: :money) if policy(marketplace).edit? %>
-      <%= render button({child: :products}, icon: :tag)  if policy(marketplace.products).index? %>
-      <%= render button({child: :delivery_areas}, icon: :map)  if policy(marketplace.delivery_areas).index? %>
-      <%= render button({child: :tax_rates}, icon: :receipt_percent)  if policy(marketplace.tax_rates).index? %>
-      <%= render button({child: :orders}, icon: :cart)  if policy(marketplace.orders).index? %>
-      <%= render button({child: :notification_methods}, icon: :bell)  if policy(marketplace.notification_methods).index? %>
-    </nav>
+<div class="flex flex-col space-y-3">
```

Comment:
This is more verbose but inverts responsibility for _layout_ styling to the parent, which generally preferred since it's easier to reason about child(ren) layout from the top level.

Reply:
Does this mean we need to find the places we're using the `CardComponent` and make sure we add a section or div that determines their layout?

If so, can we make that an independent PR?